avaframe/in3Utils/generateTopo.py

```diff
@@ -676,6 +676,11 @@ def generateTopo(cfg, avalancheDir):
     # If a drop shall be introduced
     if cfg['TOPO'].getboolean('drop'):
         z = addDrop(cfg, x, y, z)
+    
+    # moves topo in z direction 
+    if cfg['DEMDATA']['zEdit'] != '':
+    	z = z+cfg['DEMDATA'].getfloat('zEdit')
+    	log.info('Changed topo elevation by zEdit')
```

Comment:
If you want you could give the value for zEdit as well, might be helpful...

But that's only if you want to...

Reply:
I think that is a good idea- so you could change it to: log.info('Changed topo elevation by %.2f' % cfg['DEMDATA'].getfloat('zEdit'))